### PR TITLE
fix[docs] Documentation Best Practices

### PR DESCRIPTION
**Pull request by valasek**
**What kind of change does this PR introduce?** (check at least one)

- [ x] Bugfix
- [ ] Feature
- [ x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x ] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by valasek** [comment]
@smolinari, @hawkeye64 please do the code review and confirm so it can be merged.

File: docs/src/pages/contribution-guide/contribution-guide.md [hunk 1]
```diff
@@ -111,11 +111,23 @@ If your comment simply reads "+1", then odds are that other reviewers aren't goi
...
 
+### Documentation Best Practices
+
+Over time we consolidated a set of rules which we follow and following them will speed up the merge process:
+- Capitalize titles, see [How to Use Capitalize My Title](https://capitalizemytitle.com/).
+- Use the present tense.
+- Be concise, avoid text / code duplication.
+- Link to the external sources which are used as master information sources and are usually updated more frequently, like [Mozilla MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript), [Vue.js API](https://vuejs.org/v2/api/) rather than compiled tutorials which tend to be outdated soon.
+- Do the proofreading before opening a PR
+- Do not repeat texts from other sources, but repeat keep only things which are relevant and shows in a context Quasar specific features
```
Comment (lucasfernog) [inline docs/src/pages/contribution-guide/contribution-guide.md]: "but keep"

Comment (valasek) [reply docs/src/pages/contribution-guide/contribution-guide.md]: Thanks. Fixed

**Comment by rstoenescu** [comment]
Nice work.

**Pull request merged**